src/gen/flags.rs

```diff
@@ -0,0 +1,8 @@
+// flags controlling to condition parsing
+
+// in conditions output, integers must use canonical encoding (i.e. no redundant
+// leading zeros)
+pub const COND_CANON_INTS: u32 = 0x1000;
```

Comment:
You could probably dump these flags into `run_gen`.

Reply:
they don't really belong to the python binding either though.